app/src/hooks/useSafeApp.tsx

```diff
@@ -10,7 +9,7 @@ export const useSafeApp = () => {
   useEffect(() => {
     if (!safeInfo) {
       safeSdk.appsSdk.addListeners({
-        onSafeInfo: (safeInfo: SafeInfo) => setSafeInfo(safeInfo),
+        onSafeInfo: safeInfo => setSafeInfo(safeInfo),
```

Comment:
@germartinez I'm also not sure about this LOC. New safe apps SDK doesn't use event-driven architecture anymore, not sure if it will work

Reply:
`contract-proxy-kit` `v2.2.0-alpha.1` is using `safe-apps-sdk` `v0.4.1`. It should be updated when the next CPK release comes out :)

Reply:
Omen is using two versions of `safe-apps-sdk` in this PR, one that uses the old event-driven architecture that comes with `contract-proxy-kit` and the other to send transactions with safeTxGas support through `safe-apps-sdk` v1.0.1. This is only until `contract-proxy-kit` supports v1+ of `safe-apps-sdk` 👍